.travis.yml

```diff
@@ -50,6 +50,10 @@ before_script:
   # Remember the Drupal installation path.
   - DRUPAL_DIR=$(pwd)/drupal
 
+  # Download Drupal console so we can run the test for it. Skip this for the
+  # coding standards test.
+  - test ${TEST_SUITE} == "PHP_CodeSniffer" || composer require symfony/dom-crawler:2.8.11 drupal/console-core:1.0.0-rc5 drupal/console:1.0.0-rc5 --working-dir=$DRUPAL_DIR
```

Comment:
Who is talking about 0.11.3? I suggested to not hardcode the version but use the standard way of doing it, specifying a minimum version, like this:

```
$ composer require drupal/console:~1.0.0-rc5
```